packages/main/src/TableCell.ts

```diff
@@ -30,6 +31,15 @@ import { LABEL_COLON } from "./generated/i18n/i18n-defaults.js";
 	template: TableCellTemplate,
 })
 class TableCell extends TableCellBase {
+	onBeforeRendering() {
+		super.onBeforeRendering();
+		if (this.horizontalAlign && Object.values(TableCellHorizontalAlign).includes(this.horizontalAlign as TableCellHorizontalAlign)) {
```

Comment:
again i guess 2nd part of the check is not necessary. If they write "Zentrum" instead of "Center" this is their problem.